runbotics/runbotics-scheduler/src/websocket/bot/bot.gateway.ts

```diff
@@ -62,12 +62,25 @@ export class BotWebSocketGateway implements OnGatewayDisconnect, OnGatewayConnec
         this.logger.log(`Bot disconnected: ${installationId} | ${client.id}`);
     }
 
+    @UseGuards(WsBotJwtGuard)
+    @SubscribeMessage(BotWsMessage.RUNNING_PROCESS)
+    async runningProcessListener( 
+        @ConnectedSocket() socket: BotAuthSocket,
+        @MessageBody() busyBotStatus: BotStatus.BUSY,
+    ) {
+        this.logger.log('<= Process is already running');
+        this.changeBotStatusToBusy(socket.bot, busyBotStatus);
```

Comment:
to avoid "to" linking word i suggest rename changeBotStatusToBusy to setBotStatusBusy

Reply:
won't scheduler react and stop process run when bot status is changing to busy?